frontend/src/components/login/Login.component.js

```diff
@@ -125,6 +125,70 @@ const Login = () => {
             </button>
           </div>
         </div>
+      </div> */}
+
+
+      <div className="min-h-full min-w-full bg-pixelTrain bg-cover pixel-font">
+        <form
+          className=""
+          onSubmit={handleSubmit}
+          data-test="loginForm"
+        >
+          <div className="h-screen flex justify-center items-center">
+            <div className="m-7 flex flex-col relative w-2/5">
+              <div className="bg-japanese-brown mt-7 shadow-2xl sm:rounded-xl">
+                <div className="p-5 text-white">
+                  <h1 className="text-8xl md:text-5xl lg:text-7xl">.</h1>
+                  <p className="text-base md:text-lg lg:text-8xl">.</p>
+                  <p className="sm:mt-32 text-base md:text-lg lg:text-7xl">.</p>
+                </div>
+              </div>
+              <input type="hidden" name="remember" defaultValue="true" />
```

Comment:
What is this input for?

Reply:
it was from the previous code design, I didn't know what it did so I copied it over into the new design.